packages/perseus/src/widgets/video/video.tsx

```diff
@@ -77,6 +78,12 @@ class Video extends React.Component<Props> implements Widget {
             url = url.replace("{host}", embedHostname);
         }
```

Comment:
Same question here, we have some very specific logic about where to load embedded videos from. Should this move to the client?